core/common/src/main/java/alluxio/Constants.java

```diff
@@ -73,7 +73,6 @@ public final class Constants {
   public static final String HEADER_HTTPS = "https://";
   public static final String HEADER_KODO = "kodo://";
   public static final String HEADER_OSS = "oss://";
-  public static final String HEADER_OZONE = "o3fs://";
```

Comment:
OZONE's schema needs to be configured through the PropertyKey, because OZONE's schema may have multiple different prefixes, just like `PropertyKey UNDERFS_HDFS_PREFIXES`